pkg/releasenotes/notes/use-new-serialization-for-events-and-servicechecks-2b9d2f1331444637.yaml

```diff
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    Fix 'Error code "413 Request Entity Too Large" received while sending transaction to "https://dd.agent.datadoghq.eu/api/v1/check_run?api_key=*************************28e95": Payload too big, dropping it'
```

Comment:
This is not a very clear message for user to see in the changelog. We should better decribe the issue and th fix I think.

What about somethinf like:

```
Add options to enable stream serialization for `events` and `service_check` in order to fix error 'Request Entity Too Large' when sending payload to Datadog.
```
?

Reply:
The Trello card was created after a customer issue who had 'Error code "413 Request Entity Too Large'. Should I add two lines, one as fixe and another one as improvement or one is enough?
What do you think?